src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {
+                new Truck("truck"),
+                new Bulldozer("bulldozer"),
+                new Excavator("excavator")
```

Comment:
According to the checklist, we should use abstract references instead of specific ones to allow for polymorphism. Consider using `Machine truck = new Truck("truck");` instead of `new Truck("truck")`, and similarly for the other `Machine` types.